package/gluon-core/luasrc/lib/gluon/upgrade/020-interfaces

```diff
@@ -78,9 +78,6 @@ end
 uci:delete('network', 'lan')
 uci:delete('network', 'wan')
```

Comment:
does this boughs are not already included in 'device' ?

Reply:
These are not device but interface sections:
`network.lan=interface`

Reply:
ah, yes sure